src/generator.js

```diff
@@ -491,13 +574,25 @@ class Generator {
           example: node.example,
           extensionPrefix: extensionPrefix
         };
+
         node.domainIncludes.forEach(prop => {
-          let model = this.models[prop];
+          let model;
+          if (extension.preferOA) {
+            model =
+              this.models[this.getPropNameFromFQP(prop)] || this.models[prop];
+          } else {
+            model = this.models[prop];
+          }
+
           if (model) {
             model.extensionFields = model.extensionFields || [];
             model.fields = model.fields || {};
             model.extensionFields.push(field.fieldName);
             model.fields[field.fieldName] = field;
+          } else {
+            console.error(
+              `*** couldn't attach property "${field.fieldName}" onto "${prop}"`
```

Comment:
Noting something you were saying about documenting the odd properties from schema.org which has domains including classes not in plain schema.org (e.g pagination in schema.org includes Chapter in it's domain but Chapter is defined by bib.schema.org) 🤷‍♂